### DISCUSSION TITLE
Asset function or role – initially T, but it will be desirable to create codes for this – C

### DISCUSSION THREAD

**Discussion by luke-feeney** [CriticalAssetManagement/CAMS#116] [General]
We need to further clarify what this is all about and how we should think about implementation

https://github.com/CriticalAssetManagement/CAMS/issues/30

**Comment by luke-feeney** [comment]
@rpwilliams1958 @ErikHsu @GavinMendelGleason @KittyJose

**Comment by sreyac** [comment]
Settle with text for now
